django-website/portfolio/templates/portfolio/portfolio_page.html

```diff
@@ -11,15 +11,17 @@ <h1 class="section_heading">{{ page.section_title|richtext }}</h1>
   <div class="section-subtitle js-show_subtitle">{{ page.section_subtitle|richtext }}</div>
   <div class="works-container">
   {% for project in page.projects.all %}
-    <div class="work js-reveal_article">
-      <div class="work-info -u-element_to_reveal">
-        <h4 class="work-info-title">{{ project.name }}</h4>
-        <h5 class="work-info-type">{{ project.category }}</h5>
-        <div class="work-info-description">{{ project.description|richtext }}</div>
-        {% if project.link %}
-          <a class="button glow_link work-info-button" href="{{ project.link }}" target="_blank">Visit the case</a>
-        {% endif %}
-      </div>
+    {% if not project.show_in_home %}
+      <div class="work js-reveal_article">
+        <div class="work-info -u-element_to_reveal">
+          <h4 class="work-info-title">{{ project.name }}</h4>
+          <h5 class="work-info-type">{{ project.category }}</h5>
+          <div class="work-info-description">{{ project.description|richtext }}</div>
+          {% if project.link %}
+            <a class="button glow_link work-info-button" href="{{ project.link }}" target="_blank">Visit the case</a>
+          {% endif %}
+        </div>
+       {% endif %}
```

Comment:
If statement seems to be closed in the wrong place. Should not be closed after the `work` div?